packages/block-library/src/button/edit.js

```diff
@@ -225,6 +227,7 @@ function ButtonEdit( props ) {
 							? borderRadius + 'px'
 							: undefined,
 						...colorProps.style,
+						...spacingProps.style,
```

Comment:
Should this be ported to [edit.native.js](https://github.com/WordPress/gutenberg/blob/e85cfce55009f01c223faa149ad5f19e5f29c29e/packages/block-library/src/button/edit.native.js#L454) as well? Asking because I don't know :) and I can't yet get the native mobile test env running locally.

Reply:
So I got it running and saw that we do have button props. Color and spacing aren't there. Maybe for a follow up PR?

<img width="510" alt="Screen Shot 2021-05-20 at 1 54 04 pm" src="https://user-images.githubusercontent.com/6458278/118916772-f7383b80-b972-11eb-9211-b0c861f12bbf.png">